File: src/server/gkill/api/req_res/get_updated_datas_by_time_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
)

type GetUpdatedDatasByTimeResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	UpdatedIDs []string `json:"updated_ids"`
}

File: src/client/classes/api/req_res/get-updated-datas-by-time-response.ts
'use strict'

import { GkillAPIResponse } from '../gkill-api-response'

export class GetUpdatedDatasByTimeResponse extends GkillAPIResponse {

    updated_ids: Array<string>

    constructor() {
        super()
        this.updated_ids = new Array<string>()
    }

}


